lib/template/address-json.php

```diff
@@ -57,4 +57,8 @@
     }
 }
 
-javascript_renderData($aFilteredPlaces);
+if ($sOutputFormat == 'geojson' && !isset($aFilteredPlaces['error'])) {
+    javascript_renderData(jsonResultsToGeoJson([$aFilteredPlaces]));
```

Comment:
This is a very clever idea but I'm a bit concerned that now the data gets copied around yet another time.

Reply:
I started with having everything in the `search-json.php` template but the various conditions became a mess and a lot harder to test. Later I moved the logic into `Lib.php`

Reply:
What I meant is that I would prefer an extra template instead of transforming the json template yet again.